content/src/content/jcr_root/apps/core/wcm/components/text/v2/text/_cq_design_dialog/.content.xml

```diff
@@ -257,10 +257,10 @@
                                                         <items jcr:primaryType="nt:unstructured">
                                                             <links
                                                                 jcr:primaryType="nt:unstructured"
-                                                                jcr:title="Show link insertion options (link, unlink)"
+                                                                jcr:title="Show link insertion options (link, unlink, anchor)"
                                                                 sling:resourceType="cq/gui/components/authoring/dialog/inplaceediting/configuration/plugin"
                                                                 defaultEnabled="{Boolean}true"
-                                                                features="modifylink,unlink"
+                                                                features="modifylink,unlink,anchor"
```

Comment:
For existing policies, where this checkbox is already enabled, it will appear unchecked after the update, as not all 3 features are initially set, so the template author has to set the checkbox again when editing the policy after the update. This needs to be noted in the release notes